### PR TITLE
BCF-3258 - Fix QueryKey test case flakiness

### PR DESCRIPTION
**Pull request by ilija42**
TIcket: [link](https://smartcontract-it.atlassian.net/browse/BCF-3258)

Requires: https://github.com/smartcontractkit/chainlink-common/pull/577

### CODE REVIEW

**Review by EasterTheBunny — DISMISSED** [review]
LGTM

File: core/scripts/go.mod [hunk 1]
```diff
@@ -24,7 +24,7 @@ require (
 	github.com/prometheus/client_golang v1.17.0
 	github.com/shopspring/decimal v1.3.1
 	github.com/smartcontractkit/chainlink-automation v1.0.4
-	github.com/smartcontractkit/chainlink-common v0.1.7-0.20240614134652-1fb0b48758af
+	github.com/smartcontractkit/chainlink-common v0.1.7-0.20240617160144-fa92852ab5d9
```
Comment (bolekk) [inline core/scripts/go.mod:27]: Mind bumping to latest while you're at it?

Comment (ilija42) [reply core/scripts/go.mod:27]: tidy keeps it at  this version

**Review by krehermann — DISMISSED** [review]

**Review by patrickhuie19 — DISMISSED** [review]

**Review by EasterTheBunny — DISMISSED** [review]

**Review by yongkangchia — DISMISSED** [review]
thanks for the fixes to core

**Review by george-dorin — APPROVED** [review]

**Review by yongkangchia — APPROVED** [review]

**Pull request merged**